client/plots/profilePlot.js

```diff
@@ -598,19 +605,15 @@ export function makeChartBtnMenu(holder, chartsInstance, chartType) {
 	}
 }
 
-export function getProfilePlotConfig(app, opts) {
-	const state = app.getState()
-	const activeCohort = state ? state.activeCohort : opts.activeCohort
-	const key = activeCohort == FULL_COHORT ? 'full' : 'abbrev'
-	const config = app.vocabApi.termdbConfig?.plotConfigByCohort[key]?.[opts.chartType]
-	if (!config) throw `No data available form the plot ${opts.chartType} in this dataset`
-	return config
+export function getProfilePlotConfig(state, opts) {
+	const key = state.activeCohort == FULL_COHORT ? 'full' : 'abbrev'
+	const config = state.termdbConfig?.plotConfigByCohort[key]?.[opts.chartType]
+	if (!config) throw `No data available for the plot ${opts.chartType} in this dataset`
+	return structuredClone(config)
 }
 
-export async function loadFilterTerms(config, app, opts) {
-	const state = app.getState()
-	const activeCohort = state ? state.activeCohort : opts.activeCohort //when recovering a session the state is not available, use the opts provided
-	const cohortPreffix = activeCohort == FULL_COHORT ? 'F' : 'A'
+export async function loadFilterTerms(config, state, app) {
+	const cohortPreffix = state.activeCohort == FULL_COHORT ? 'F' : 'A'
 	const twlst = []
 	config.countryTW = { id: cohortPreffix + 'country' }
 	config.regionTW = { id: cohortPreffix + 'WHO_region' }
```

Comment:
why is `cohortPrefix` needed for `tw.id`? and this is for the tw.id, not for term.id, right?